src/components/Breadcrumbs.astro

```diff
@@ -13,7 +13,7 @@ breadcrumbList[0] === "posts" &&
 ---
 
 <nav class="breadcrumb" aria-label="breadcrumb">
-  <ul>
+  <!-- <ul>
```

Comment:
Remember to remove this outcommented part. And fix nav tag error
<img width="686" alt="image" src="https://github.com/flo-kn/astro-paper/assets/104433967/66ff4609-ff23-4ff5-b11a-c577d1aea85d">